app/bundles/LeadBundle/Model/ImportModel.php

```diff
@@ -703,7 +718,7 @@ protected function dispatchEvent($action, &$entity, $isNew = false, Event $event
      * @param string $msg
      * @param Import $import
      */
-    protected function logDebug($msg, Import $import = null)
+    public function logDebug($msg, Import $import = null)
```

Comment:
Why is this function public? Do we really want to write to log from the command? Is there any other way of processing code that will not write error to the log?